docs/cloud-native-security/kspm-benchmark-rules.asciidoc

```diff
@@ -11,21 +16,21 @@ Each benchmark rule checks to see if a specific type of resource is configured a
 * `Ensure IAM policies that allow full "*:*" administrative privileges are not attached`
 * `Ensure the default namespace is not in use`
 
+When benchmark rules are evaluated, the resulting <<findings-page, findings>> data appears on the <<cloud-posture-dashboard, Cloud Security Posture dashboard>>. 
 
-When benchmark rules are evaluated, the resulting <<findings-page, findings>> data appears on the <<cloud-posture-dashboard, Cloud Security Posture dashboard>>.
+NOTE: Benchmark rules are not editable.
 
-To find the Benchmark Integrations page, go to **Rules -> Benchmark rules**. From there, you can view the benchmark rules associated with an existing integration by clicking the integration name.
+[discrete]
+== Review your benchmarks
 
-[role="screenshot"]
-image::images/benchmark-rules.png[Benchmark rules page]
+To see your active benchmarks, go to **Rules -> Benchmarks**. From there, you can click an existing security posture management integration's name to view the benchmark rules associated with it. You can click on an individual benchmark rule to see details including information about how to remediate it, and related links. 
 
-You can then click on a benchmark rule's name to see details, including information about how to remediate failures and related links.
+Benchmark rules are enabled by default, but you can disable some of them to customize an integration for your environment. To enable or disable a rule, use the **Enabled** toggle on the right of the rules table. 
 
-NOTE: Benchmark rules are not editable.
+NOTE: Disabling a benchmark rule automatically disables any associated detection rules and alerts. Re-enabling a benchmark rule does not automatically re-enable them.
```

Comment:
> Re-enabling a benchmark rule does not automatically re-enable them

what do you think about bolding the "does not"? I think the behaviour itself is a bit confusing so it might be good to emphasise the difference between disabling a rule and enabling it